api/src/main/java/org/openmrs/scheduler/Schedule.java

```diff
@@ -231,21 +226,6 @@ public void setRepeatInterval(long repeatInterval) {
 	}
 	
 	/**
-	 * Get the date format used to set the start time.
-	 */
-	public String getDateFormat() {
```

Comment:
@dkayiwa should we deprecate this first? I mean its not used in the core but what if other modules use it? Or does it suffice to put this in a changelog that from 2.2.0 they do not exist anymore